aspnetcore/security/authentication/identity-configuration.md

```diff
@@ -2,22 +2,86 @@
 title: Configure Identity
 uid: security/authentication/identity-configuration
 ---
+
 # Configure Identity
 
-ASP.NET Core Identity has some default behaviors that you can override easily in your application's startup class.
+ASP.NET Core Identity has some default behaviors that you can override easily in your application's `Startup` class.
 
 ## Passwords policy
 
-By default, Identity requires that passwords contain an uppercase character, lowercase character, and digits. There are also some other restrictions. If you want to simplify password restrictions, you can do that in the startup class of your application.
+By default, Identity requires that passwords contain an uppercase character, lowercase character, a digit, and an alphanumeric character. There are also some other restrictions. If you want to simplify password restrictions, you can do that in the `Startup` class of your application.
+
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+ASP.NET Core 2 added the `RequiredUniqueChars` options. Otherwise, the options are the same from ASP.NET Core 1.x.
+
+[!code-csharp[Main](identity/sample/src/ASPNETv2-IdentityDemo-Configuration/Startup.cs?range=29-37,50-52)]
+
+# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+
+[!code-csharp[Main](identity/sample/src/ASPET-IdentityDemo-PrimaryKeysConfig/Startup.cs?range=60-65)]
```

Comment:
This 1.x snippet should display like the following, to provide a bit more context:
```csharp
services.Configure<IdentityOptions>(options =>
{
    // Password settings
    options.Password.RequireDigit = true;
    options.Password.RequiredLength = 8;
    options.Password.RequireNonAlphanumeric = false;
    options.Password.RequireUppercase = true;
    options.Password.RequireLowercase = false;
});
```